### DISCUSSION TITLE
Bounding box for saved clips and Frigate (docker) at sidebar?

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#967] [Ask A Question]
Is it possible to store bounding box to the saved clip and is there any idea for that?

Also... addon for Home Assistant can give sidebar for Frigate... Is it possible to have Frigate at sidebar if using frigate with docker? And or what would be the best way to handle clips & stored images with HA?

**Comment by hawkeye217** [comment]
You should install the [official integration](https://github.com/blakeblackshear/frigate-hass-integration) if you are using Frigate with Docker.

**Reply by McPaHa** [reply]
I've installed that integratio and I see sensors etc but there is no sidepanel...

**Reply by paularmstrong** [reply]
The Home Assistant add-on gets the item in the sidebar. If you're running Frigate on another system, you can create a [custom iframe panel](https://www.home-assistant.io/integrations/panel_iframe/) instead.

**Reply by McPaHa** [reply]
Thx... I tried it and it worked exactly I weanted!

**Reply by McPaHa** [reply]
Hmm.. actually HA app for Android & iPhone doesn't show that iframe panel content. With pc win 10 and chrome browser it shows ok.

**Comment by ohhai-kthxbai** [comment]
+1 for bounding boxes stored with clips. It'd help a lot with understanding how/where/when objects are detected. Thanks!
